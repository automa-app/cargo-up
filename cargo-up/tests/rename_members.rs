use insta::assert_snapshot;
mod utils;

#[test]
fn test_rename_members() {
    let (out, err) = utils::run_upgrader("rename_members", "0.3.0", true);
    assert_snapshot!(out);
    assert_snapshot!(err);
}

// TODO: Member of member in Struct or Mmeber of variant in Enum
