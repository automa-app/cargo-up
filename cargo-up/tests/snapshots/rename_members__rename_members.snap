---
source: tests/rename_members.rs
expression: out
---

